### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by sau9458**

### CODE REVIEW

File: src/services/SignupServices.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  .then( (response)=> {
+    console.log(response);
+  })
+  .catch(function (error) {
+    console.log(error);
+  });
+}
+
+export const signUpApi = (data1) => {
+  return fetch(`https://secure-refuge-14993.herokuapp.com/add_user?username=${data1.username}&password=${data1.password}&role=${data1.option}`).then((response) => {
```
Comment (mageexxcel) [inline src/services/SignupServices.js:14]: base url should be config file

File: src/services/SignupServices.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import 'whatwg-fetch';
+
+export const signupapi=()=>{
+    axios.get('https://secure-refuge-14993.herokuapp.com/add_user?username=admin&password=admin&role=admin')
```
Comment (mageexxcel) [inline src/services/SignupServices.js:4]: base url should be in  config file

File: src/services/LoginServices.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+export const loginapi=()=>{
+    axios.get('https://secure-refuge-14993.herokuapp.com/login?username=admin&password=admin')
```
Comment (mageexxcel) [inline src/services/LoginServices.js:3]: base url should be in config file

File: src/modules/signUp/signUp.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    this.setState({password: e.target.value});
+  }
+
+  handleOptionChange=(e)=>{
+   this.setState({option:e.target.value})
+ }
+
+  componentWillReceiveProps(props){
+    if(props.signUp.error === 1){
+       alert('Account already exist')
```
Comment (mageexxcel) [inline src/modules/signUp/signUp.js:43]: this is a normal alert ?? change it to your generic alerts

File: src/modules/signUp/signUp.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    }
+    else if(props.signUp.error===0){
+      this.props.router.push('/login');
+      alert("you are sucessfull sign up");
```
Comment (mageexxcel) [inline src/modules/signUp/signUp.js:47]: this is a normal alert ?? change it to your generic alerts

File: src/services/LoginServices.js (hunk 3 above)
Comment (sau9458) [reply src/services/LoginServices.js:3]: ok sir

File: src/modules/signUp/signUp.js (hunk 5 above)
Comment (sau9458) [reply src/modules/signUp/signUp.js:47]: ok sir

File: src/modules/signUp/signUp.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    super(props);
+    this.state={
+      username:'',
+      password:'',
+      option:'admin'
+    }
+  };
+  onFormSubmit(e){
+    e.preventDefault();
+    var data1={
```
Comment (mageexxcel) [inline src/modules/signUp/signUp.js:21]: use let

Comment (sau9458) [reply src/modules/signUp/signUp.js:21]: ok sir

**Pull request merged**